app/javascript/pages/App/index.js

```diff
@@ -14,7 +13,6 @@ import UpdateUserOfficeMutation from './UpdateUserOfficeMutation.gql'
 
 class AppPage extends Component {
   constructor(props) {
```

Comment:
I reckon we can remove `constructor` altogether, since it's not doing anything other than calling `super(props)`.

Reply:
Haha yes, good catch!

Reply:
I turned it into a function component as well 🙂